docs/cryoet_data_portal_docsite_landing.md

```diff
@@ -42,20 +42,24 @@ Here is a summary of the datasets that have been provided. A full description ca
 ## S3 Bucket Info
 The CryoET Data Portal S3 bucket supports public access. The bucket URL is:
 
-    `s3://cryoet-data-portal-public`
+```
+s3://cryoet-data-portal-public
+```
 
-  To list the bucket contents with the S3 CLI without credentials, please use the following:
+To list the bucket contents with the S3 CLI without credentials, please use the following:
 
-    `aws s3 ls --no-sign-request s3://cryoet-data-portal-public`
+```
+aws s3 ls --no-sign-request s3://cryoet-data-portal-public
+```
   
 ## Citing the CryoET Data Portal
  
 Data from the portal must acknowledge the data providers and the original publications as follows (inserting of omitting as needed based on which datasets are used):
-Some of the data used in this work was provided by the group (s) of Julia Mahamid (EMBL)/Juergen Plitzko (MPI). The work is described more fully in the publication:
+Some of the data used in this work was provided by the group (s) of Julia Mahamid (EMBL)/Jürgen Plitzko (MPI). The work is described more fully in the publication:
```

Comment:
Consistency with the umlaut spelling elsewhere on this page

Reply:
Also, in the line just above this, it should be inserting *or* omitting.

Reply:
> Also, there's a typo in the line just above this: (of -> or)
> 
> > Data from the portal must acknowledge the data providers and the original publications as follows (inserting of omitting as needed based on which datasets are used):
> 
> > Data from the portal must acknowledge the data providers and the original publications as follows (inserting **or** omitting as needed based on which datasets are used):

Fixed.